core/src/main/scala/org/apache/spark/deploy/security/ConfigurableCredentialManager.scala

```diff
@@ -41,15 +41,17 @@ import org.apache.spark.util.Utils
  * For example, Hive's credential provider [[HiveCredentialProvider]] can be enabled/disabled by
  * the configuration spark.yarn.security.credentials.hive.enabled.
  */
-private[yarn] final class ConfigurableCredentialManager(
+private[spark] class ConfigurableCredentialManager(
     sparkConf: SparkConf, hadoopConf: Configuration) extends Logging {
   private val deprecatedProviderEnabledConfig = "spark.yarn.security.tokens.%s.enabled"
```

Comment:
Shall we also rename the configurations, currently it is "spark.yarn.security.*"?